### ISSUE TITLE
[eslint-plugin-lit-a11y] no-invalid-change-handler should be deprecated

### ISSUE THREAD

**Issue by geoffrich** [open-wc/open-wc#2218]
## Expected behavior

The no-onchange rule has been [deprecated](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/master/docs/rules/no-onchange.md) in eslint-plugin-jsx-a11y as of Nov 2020. The corresponding rule (no-invalid-change-handler) should be deprecated and turned off in the recommended lit-a11y config.

The following code should not report an error with the recommended config:

```js
html`<select @change=${foo}></select>`
```

## Actual Behavior
It is turned on in the recommended lit-a11y config and is not deprecated.

The above code reports the following error: 

> blur must be used instead of change on `<select>`, unless absolutely necessary and it causes no negative consequences for keyboard only or screen reader users. eslintlit-a11y/no-invalid-change-handler

## Additional context

eslint-plugin-lit-a11y v1.0.1

The original reasoning behind this rule seemed to be that IE fired the change event when navigating between options with the arrow keys. From the [linked article](https://www.themaninblue.com/writing/perspective/2004/10/19/) in the jsx-a11y docs:

> The only problem is, when you use the keyboard to explore the options in a select menu (using cursor keys) it triggers the onchange event handler (in IE). So, if you have any actions that rely on that handler, they'll probably be executed before the user has time to listen to all the options in the select menu. It'll also mean they'll probably only ever be able to access the first item. 

This was from 2004 and seems to be out of date now. I wrote a [quick CodePen](https://codepen.io/geoff-rich/full/VwKxxaw) to see when the change event is fired for a `<select>` and tested in IE11, Chrome, and Firefox. In all browsers tested, tabbing to the select, opening the menu with Space, and using the arrow keys to navigate between different items did **not** fire a change event. A change event was only fired when an item was selected with Enter or if the arrow keys were used to select an option without opening the menu.

Since Lit targets modern browsers, legacy (<IE11) select behavior should not be an a11y concern.

**Comment by geoffrich** [comment]
If the maintainers agree on deprecating this rule, I'm happy to work on a PR for it.

**Comment by Westbrook** [comment]
Sounds good to me. Happy to accept a PR here. I’d venture to guess some of the realities around the way React coalesces `input` and `change` events, which we don’t do anyways 😉

Thanks for bringing this to our attention!

**Comment by bennypowers** [comment]
Agreed, looking forward to review the PR
